src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,25 @@
+import { ChangeEvent } from "react"
+import "./Slider.css"
+
+export interface SliderProps {
+  min: number
+  max: number
+  onChange: (E: ChangeEvent<HTMLInputElement>) => void
+  value: number
+}
+
+export function Slider({ min, max, onChange, value }: SliderProps) {
+  return (
+    <section className="range">
```

Comment:
[nit] I probably would just make this a div. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section. MDN states if a wrapper is used as a styling element, it should usually be a div. 

"If you are only using the element as a styling wrapper, use a [<div>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div). An unwritten rule is that a <section> should logically appear in the outline of a document."